arangod/Transaction/Manager.cpp

```diff
@@ -346,6 +346,9 @@ Result Manager::createManagedTrx(TRI_vocbase_t& vocbase, TRI_voc_tid_t tid,
     return res.reset(TRI_ERROR_BAD_PARAMETER,
                      "<lockTimeout> needs to be positive");
   }
+  if (isFollowerTransaction) {
```

Comment:
```suggestion
  if (isFollowerTransaction && ServerState::instance()->isDBServer()) {
```

Reply:
I had to revert these changes since in the unit tests the server does not have role DBServer. :-(